packages/core/src/browser/view-container.ts

```diff
@@ -330,7 +330,7 @@ export class ViewContainer extends BaseWidget implements StatefulWidget, Applica
     }
 
     get containerLayout(): ViewContainerLayout {
-        return this.panel.layout as ViewContainerLayout;
```

Comment:
I don't understand how it can be `null`: `init` is called before any other call and it sets panel and its layout.